src/or/channeltls.h

```diff
@@ -12,6 +12,8 @@
 #include "or.h"
 #include "channel.h"
 
+#define TLS_PER_CELL_OVERHEAD 29
+
```

Comment:
Once we get a hang on pastly, would be good to have a comment explaining that value. Lets keep that in mind.